### PR TITLE
Compare both the IDs as same data types when comparing

### PR DESCRIPTION
**Pull request by zip15**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -32,9 +32,9 @@
 @client.event
 async def on_ready():
     for user in user_list:
-        if int(user) in ignored_channels_list:
```
Comment (arnavgpt) [inline main.py:35]: this is required so that if the user ID is a string, it is converted and checked

Comment (zip15) [reply main.py:35]: weird, it wasnt skipping ids in the ignore list until i changed it, but alright

Comment (zip15) [reply main.py:35]: yeah, just tested and it isnt skipping ids in the list when you do that.

Comment (arnavgpt) [reply main.py:35]: Did you added channels ID as strings or Integer?

**Comment by arnavgpt** [comment]
I will make a check that whenever both the IDs are compared, they are in same data type

**Comment by arnavgpt** [comment]
It Looks like you have deleted your fork so I couldn't edit. I will make a commit instead

**Comment by arnavgpt** [comment]
Fixed in the [commit](https://github.com/Empyreann/discord-dms-opener/commit/79a39907dd547f3865e1149c50e02b8f24cb4390)
Once again thanks for bringing this up

**Pull request closed without merging**